hive-metastore/src/main/java/org/apache/iceberg/hive/HiveSchemaConverter.java

```diff
@@ -99,6 +99,10 @@ Type convertType(TypeInfo typeInfo) {
           case INTERVAL_YEAR_MONTH:
           case INTERVAL_DAY_TIME:
           default:
+            // special case for Timestamp with Local TZ which is only available in Hive3
+            if ("TIMESTAMPLOCALTZ".equals(((PrimitiveTypeInfo) typeInfo).getPrimitiveCategory().name())) {
```

Comment:
`equalsIgnoreCase`?

Reply:
I think it should always be all uppercase, but I've changed it just to be safe.